lane.py

```diff
@@ -1994,12 +2004,13 @@ def _done_with_lane(lane):
                 # Lane at all. Do a whole separate query and plug it
                 # in at this point.
                 for x in lane.groups(
-                    _db, include_sublanes=False, facets=facets,
+                    _db, include_sublanes=False,
+                        pagination=pagination, facets=facets,
                 ):
                     yield x
 
     def _featured_works_with_lanes(
-        self, _db, lanes, facets, pagination, search_engine, debug=False
+        self, _db, lanes, pagination, facets, search_engine, debug=False
```

Comment:
This makes the method signatures consistent -- pagination always comes before facets.